docs/source/_static/css/custom.css

```
@@ -41,6 +41,11 @@ em.sig-param > span:not(:nth-child(1)):not(:nth-child(2)) {
   border-left: solid 3px #6ab0de;
 }
 
+/* Main content */
+.wy-nav-content {
+    max-width: 1200px;
```

A nit, but could you use two whitespaces for indent? The current code format is introduced by #1463 using https://prettier.io, and I think we need to discuss the formatting rule for HTML/CSS if we change it.
```suggestion
  max-width: 1200px;
```

---

Ops, should've noticed that. Thanks, applied.